bokehjs/src/coffee/core/bokeh_events.ts

```diff
@@ -0,0 +1,217 @@
+import { logger } from "./logging"
+
+export class Event {
```

Comment:
This needs to be renamed to `BokehEvent` (or something), so that it doesn't conflict with DOM `Event`.